aion/web/controllers/ranking_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Aion.RankingController do
+  use Aion.Web, :controller
+  alias Guardian.Plug
+  alias Aion.Ranking
+
+  plug Plug.EnsureAuthenticated, handler: __MODULE__
+
+  def ranking(conn, _params) do
```

Comment:
I guess the default idea for  `ranking`  endpoint is to display this thing, so maybe it could be named `index`. Or if not, consider putting verbs to the function names like `get_general_ranking` or whatever to signal what you are actually doing (here you actually render stuff, right?)

Reply:
I don't like the idea of having names of controllers functions starting with get_*.

Reply:
Okay, what about the first sentence from my comment?

Reply:
For now i will leave it. My next area of focus is ranking per each category. Then I will probably introduce different naming ;)